content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Quantity
+id: quantity
+date: 2023-01-24
+short_description: >
+ Quantity refers to a special notation which are portable across machines. 
```

Comment:
```suggestion
 Quantity refers to a special notation that is portable across machines. 
```

Reply:
Imo, I disagree that quantity is a "special" notation
Let's start with the comments that describes the Quantity struct
"Quantity is a fixed-point representation of a number"
The comments in [quantity.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go) describes quantity in great detail but may not be "user-friendly" enough for kubernetes.io but it's a good place to start